### PR TITLE
Add functional tests for fusion_region, fusion_sc, fusion_se, fusion_ss, fusion_tenant, fusion_ts

### PR DESCRIPTION
**Pull request by andreepyro**
##### SUMMARY
Add functional tests.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- tests

##### ADDITIONAL INFORMATION
Adding functional tests for fusion_region, fusion_sc, fusion_se, fusion_ss, fusion_tenant, and fusion_ts. Replacing old tests for fusion_array and fusion_az.

### CODE REVIEW

**Review by sdodsley — DISMISSED** [review]

File: plugins/modules/fusion_sc.py [hunk 1]
```diff
@@ -145,7 +142,7 @@ def create_sc(module, fusion):
     if not module.params["size_limit"]:
         module.params["size_limit"] = "4P"
     if not module.params["iops_limit"]:
-        module.params["iops_limit"] = "100000000"
+        module.params["iops_limit"] = "10000000"
```
Comment (jankodera-ps) [inline plugins/modules/fusion_sc.py:145]: The limit was correct

File: plugins/modules/fusion_sc.py [hunk 2]
```diff
@@ -155,11 +152,11 @@ def create_sc(module, fusion):
         )
     )
     bw_limit = parse_number_with_metric_suffix(module, module.params["bw_limit"])
-    if bw_limit not in range(1048576, 549755813889):  # 1MB/s to 512GB/s
+    if bw_limit < 1048576 or bw_limit > 549755813888:  # 1MB/s to 512GB/s
         module.fail_json(msg="Bandwidth limit is not within the required range")
-    if 100 > iops_limit > 10000000:
+    if iops_limit < 100 or iops_limit > 10000000:
```
Comment (jankodera-ps) [inline plugins/modules/fusion_sc.py:157]: The iops limit is 100,000,000

File: tests/integration/inventory [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[testgroup]
+testhost ansible_connection="local" ansible_pipelining="yes" ansible_python_interpreter="/Users/apajtas/miniconda3/bin/python"
```
Comment (jankodera-ps) [inline tests/integration/inventory:2]: This local path probably shouldn't be there. Should it be `/usr/bin/python3`, or not even there if Ansible is able to cope?

File: plugins/modules/fusion_sc.py (hunk 2 above)
Comment (jankodera-ps) [reply plugins/modules/fusion_sc.py:157]: (as a minor note, Python ignores underscores in numeric literals, so we could use them to separate magnitudes)

**Review by DenDen039 — APPROVED** [review]

File: plugins/modules/fusion_sc.py (hunk 2 above)
Comment (andreepyro) [reply plugins/modules/fusion_sc.py:157]: Thanks for noticing!

File: tests/integration/inventory (hunk 3 above)
Comment (andreepyro) [reply tests/integration/inventory:2]: You're right, shouldn't be there 👍

**Comment by andreepyro** [comment]
Resolved all comment.

**Comment by andreepyro** [comment]
@jankodera-ps please approve if you're happy with the resolution

**Review by jankodera-ps — APPROVED** [review]

**Pull request merged**